hosting/index-nix-hour.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
Maybe use nix as the shebang? This way you could make sure all deps are present.

Reply:
My preferred solution would be a `shell.nix`  for all nix-hour related tooling and packaging my script with `writeShellApplication`. However, I don't want to make that call and I only see it as a temporary stand-in for Yorik's wonderful work.